### DISCUSSION TITLE
Teable [airtable alternative]

### DISCUSSION THREAD

**Discussion by am-monkey** [coollabsio/coolify#3415] [Service Template Requests]
Airtable alternative

Seems very perspective and powerfull

Official repository
https://github.com/teableio/teable

Docker compose guide
https://help.teable.io/deployment/docker-compose#ce

**Comment by romainds-tech** [comment]
Really cool project, up to be available in coolify ! 🚀

**Comment by shahzeb1** [comment]
Here's the [Docker compose](https://github.com/teableio/teable/blob/develop/dockers/examples/standalone/docker-compose.yaml) for anyone that's looking.

**Reply by Plaidmustache** [reply]
> Here's the [Docker compose](https://github.com/teableio/teable/blob/develop/dockers/examples/standalone/docker-compose.yaml) for anyone that's looking.

were you able to get this deployed?  It does not seem to finish deploying for me. ive been trying about 100 different ways. the teable container does not start, the other will but the db migrations stays unhealthy, and so the main app does not run.

**Reply by shahzeb1** [reply]
I didn't attempt to deploy unfortunately.

If you try running it from just the teable git folder (not using coolify) does that work? (Meaning is teable the issue or Coolify)?
